implementation/eOrdering/README.txt

```diff
@@ -0,0 +1,21 @@
+Description: The eOrdering module contains classes specific to the Ordering phase of procurement. 
+
+Files:
+The conventions_report file contains the UML Conventions Comformance Report of eOrdering (as an HTML page). 
+An automatically generated report that states the UML conformance violations of the UML model in file 
```

Comment:
Not all visitors will be familiar with the UML conventions, so I would add a link. Instead of  `that states the UML conformance violations of the UML model`, one can explain that the HTML lists what does not conform against the [EPO UML conventions](https://meaningfy-ws.github.io/model2owl-docs/public-review/uml/conceptual-model-conventions.html)

Reply:
Noted and much apreciated.